packages/yourloops/lib/diabeloop-url.tsx

```diff
@@ -59,33 +54,29 @@ class DiabeloopUrl {
     return this.termsUrl;
   }
 
-  getTermsLink(currentLangue: string): JSX.Element {
-    return (
-      <Link
-        href={this.getTermsUrL(currentLangue)}
-        target="_blank"
-        rel="noreferrer">
-        anyword
-      </Link>
-    );
-  }
-
   getPrivacyPolicyUrL(currentLangue: string): string {
     this.privacyPolicyUrl = `${this.rootPathName}yourloops-data-privacy.${currentLangue}.pdf`;
     return this.privacyPolicyUrl;
   }
 
-  getPrivacyPolicyLink(currentLangue: string): JSX.Element {
-    return (
-      <Link href={this.getPrivacyPolicyUrL(currentLangue)} target="_blank" rel="noreferrer">
-        anyword
-      </Link>
-    );
+  getIntendedUseUrL(currentLangue: string): string {
+    return `${this.rootPathName}intended-use.${currentLangue}.pdf`;
   }
 
-  getIntendedUseUrL(currentLangue: string): string {
-    this.intendedUseUrL = `${this.rootPathName}intended-use.${currentLangue}.pdf`;
-    return this.intendedUseUrL;
+  getLoginTrainingUrL(currentLangue: string): string {
+    return `${this.rootPathName}yourloops-login-training.${currentLangue}.pdf`;
+  }
+
+  getPatientTrainingUrL(currentLangue: string): string {
+    return `${this.rootPathName}yourloops-patient-training.${currentLangue}.pdf`;
+  }
+
+  getHcpTrainingUrL(currentLangue: string): string {
+    return `${this.rootPathName}yourloops-hcp-training.${currentLangue}.pdf`;
+  }
+
+  getCaregiverTrainingUrL(currentLangue: string): string {
+    return `${this.rootPathName}yourloops-hcp-training.${currentLangue}.pdf`;
```

Comment:
We could have just one method getTrainingUrl with a "role" argument  ? (and default to login training url if role is not in hcp/caregiver/patient )